alacritty/src/config/bindings.rs

```diff
@@ -546,7 +546,7 @@ fn common_keybindings() -> Vec<KeyBinding> {
     bindings!(
         KeyBinding;
         "v",    ModifiersState::CONTROL | ModifiersState::SHIFT, ~BindingMode::VI;                       Action::Paste;
-        "v",    ModifiersState::CONTROL | ModifiersState::SHIFT, +BindingMode::SEARCH;                   Action::Paste;
+        "v",    ModifiersState::CONTROL | ModifiersState::SHIFT, +BindingMode::VI, +BindingMode::SEARCH; Action::Paste;
```

Comment:
You should do the same for all other bindings `Paste` bindings, as well as document them in the `alacritty-binding.5.scd` man page.